src/components/interior-left-nav/interior-left-nav.js

```diff
@@ -222,6 +225,7 @@ class InteriorLeftNav extends mixin(createComponent, initComponentBySearch) {
     selectorLeftNavListItemHasChildren:
       '[data-interior-left-nav-with-children]',
     selectorLeftNavCollapse: '[data-interior-left-nav-collapse]',
+    selectorLeftNavArrow: '[data-interior-left-nav-arrow]',
```

Comment:
Probably we’d make it `[data-interior-left-nav-arrow] title` (and rename this property to `selectorLeftNavArrowTitle`). In this way, we can consolidate two qSA calls to one.